frontend/src/styles/base/_icons.scss

```diff
@@ -1,9 +1,23 @@
-@mixin icon {
+%icon {
   align-items: center;
   display: inline-flex;
   justify-content: center;
-  font-size: 1.5rem;
+}
+
+@mixin icon {
```

Comment:
Was thinking if we should take this opportunity to refactor this to a placeholder class instead since `@mixin icon` does not have any dynamic behaviour. 

See previous discussion [here](https://github.com/opengovsg/postmangovsg/pull/800#discussion_r517085746) on mixins vs placeholder class.  I think I missed out on this refactor the last time round.

Reply:
Thanks for sharing, learnt about the recommended use for mixins vs placeholders/extend!